text/0000-crates.io-default-ranking.md

```diff
@@ -140,78 +140,59 @@ By far, the most common attribute people said they considered in the survey was
 whether a crate had good documentation. Frequently mentioned when discussing
 documentation was the desire to quickly find an example of how to use the crate.
 
-- Percentage of top-level items that have documentation
-  - We have created a proof-of-concept [cargo doc-coverage][] tool to count the
-    number of public items and the percentage of those that have/don't have
-    documentation. The overall documentation coverage didn't match our human
-    perceptions of well-documentedness from looking at the front page of
-    documentation, so we decided top-level items are more important than items
-    in submodules. For example, nom is 48% documented overall, but the
-    top-level items are extremely well documented, 170/195 or 87%. Our
-    definition of "top-level" counts the overall crate as an item. We think our
-    doc coverage POC can be modified to report this number.
-  - Would need to unpack and run this on each package version in a background
-    job started by a publish; then save the percentage in crates.io's database.
-
-- In the crate root documentation, presence of a section headed with the word
-  "Example" and containing a codeblock
-  - Existing issue, seen in the survey results is that people look in both the
-    README of the repo and the front page of the docs for examples. We have an
-    opportunity to encourage at least one to be present reliably.
-  - Increases the doc percentage score by 5%
-
-- Presence of files in `/examples`
-  - Future improvement: [render and link to examples in documentation][examples]
-  - Increases the doc percentage score by 5%
-
-[cargo doc-coverage]: https://crates.io/crates/cargo-doc-coverage
-[examples]: https://github.com/rust-lang/cargo/issues/2760
-
-<table>
-    <tr>
-        <th>Crate</th>
-
-        <th>Doc coverage of top-level items</th>
-        <th>Example in the crate root docs bonus</th>
-        <th>`/examples` bonus</th>
-        <th>Overall Ease of Use score</th>
-    </tr>
-    <tr>
-        <td>peresil</td>
-        <td>10/10, 100%</td>
-        <td>5%</td>
-        <td>0%</td>
-        <td>105%, ☀️</td>
-    </tr>
-    <tr>
-        <td>combine</td>
-        <td>43/44, 98%</td>
-        <td>5%</td>
-        <td>0%</td>
-        <td>103%, ☀️</td>
-    </tr>
-    <tr>
-        <td>nom</td>
-        <td>170/195, 87%</td>
-        <td>5%</td>
-        <td>0%</td>
-        <td>92%, ☀️</td>
-    </tr>
-    <tr>
-        <td>lalrpop</td>
-        <td>4/5, 80%</td>
-        <td>0%</td>
-        <td>0%</td>
-        <td>80%, 🌤</td>
-    </tr>
-    <tr>
-        <td>peg</td>
-        <td>2/3, 66%</td>
-        <td>0%</td>
-        <td>0%</td>
-        <td>66%, 🌥</td>
-    </tr>
-</table>
+- Number of lines of documentation in Rust files:
+  `grep -r \/\/[\!\/] --binary-files=without-match --include=*.rs . | wc -l`
```

Comment:
Bit of a nit, but could the regex here please use quoting, rather than backslash-escaping every character?  That would make it much more readable.